abaplint.jsonc

```diff
@@ -38,7 +38,9 @@
     "unnecessary_chaining": false,
     "unnecessary_pragma": true,
     "7bit_ascii": true,
-    "abapdoc": true,
+    "abapdoc": {
+      "interfaceDefinition": false
+    },
```

Comment:
I guess for all other locations (except interface definitions) the check is still true by changing this setting, right?